app/src/components/HostFormDetail/HostFormGender.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useState } from 'react'
```

Comment:
@HassenH1 for the JSX elements using identical style attributes please define these variables of type `CSSProperties` 

For example, one repeated rule set apparent in this snippet could be factored out into the following reusable object:
```
const fieldContainerStyles: CSSProperties = {
   display: 'flex',
   flexDirection: 'row',
   alignItems: 'center',
   fontSize: '18px',
}
```

and used in the JSX as `<div style={fieldContainerStyles}></div>` where applicable

Reply:
Just added this